modules/lang-painless/src/main/resources/org/elasticsearch/painless/org.elasticsearch.script.fields.txt

```diff
@@ -61,6 +61,14 @@ class org.elasticsearch.script.field.ScaledFloatDocValuesField @dynamic_type {
   double get(int, double)
 }
 
+class org.elasticsearch.script.field.HalfFloatDocValuesField @dynamic_type {
+  float get(double)
```

Comment:
Can you add a note explaining why this takes a double but returns a float.

Reply:
Added.